extensions/wikia/ImageReview/ImageReviewHelper.class.php

```diff
@@ -488,13 +581,13 @@ public function getStatsData( $startYear, $startMonth, $startDay, $endYear, $end
 				while ( $row = $dbr->fetchObject( $res ) ) {
 					if ( !empty( $row->review_state ) ) {
 						if ( !isset( $data[ $reviewer ] ) ) {
-							$data[ $reviewer ] = array(
+							$data[ $reviewer ] = [
 								'name' => $user->getName(),
 								'total' => 0,
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "total" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)